zeebe/qa/integration-tests/src/test/java/io/camunda/zeebe/it/client/command/ActivateJobsTest.java

```diff
@@ -62,14 +62,41 @@ void shouldRespondActivatedJobsWhenJobsAreAvailable(
 
   @ParameterizedTest
   @ValueSource(booleans = {true, false})
-  public void shouldRespondNoActivatedJobsWhenNoJobsAvailable(final boolean useRest) {
+  public void shouldReturnEmptyListOnRequestTimeout(final boolean useRest) {
     // when
-    final CamundaFuture<ActivateJobsResponse> responseFuture =
-        getCommand(client, useRest).jobType("notExisting").maxJobsToActivate(1).send();
+    final var actual =
+        getCommand(client, useRest).jobType("notExisting").maxJobsToActivate(1).send().join();
 
     // then
-    final ActivateJobsResponse response = responseFuture.join();
-    assertThat(response.getJobs()).isEmpty();
+    assertThat(actual.getJobs()).isEmpty();
+  }
+
+  @ParameterizedTest
+  @ValueSource(booleans = {true, false})
+  public void shouldReturnEmptyListOnUserDefinedGlobalRequestTimeout(final boolean useRest) {
+    // when
+    client = zeebe.newClientBuilder().defaultRequestTimeout(Duration.ofSeconds(5)).build();
+    final var actual =
+        getCommand(client, useRest).jobType("notExisting").maxJobsToActivate(1).send().join();
+
+    // then
+    assertThat(actual.getJobs()).isEmpty();
+  }
+
+  @ParameterizedTest
+  @ValueSource(booleans = {true, false})
+  public void shouldReturnEmptyListOnUserDefinedCommandRequestTimeout(final boolean useRest) {
+    // when
+    final var actual =
+        getCommand(client, useRest)
+            .jobType("notExisting")
+            .maxJobsToActivate(1)
+            .requestTimeout(Duration.ofSeconds(5))
```

Comment:
❓ Will this make the test last 5 seconds each time? 😄 Don't get caught for wasting that much CI time!

Reply:
Yes, it blocks the test for 5 seconds :) I'll lower it to 1sec.